### PR TITLE
Fix selectors for Sql manager page

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Fix selectors for Sql manager page
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no 
| How to test?      | UI tests are Ok, maybe re-check that export action is working as expected on Sql manager page
| Possible impacts? | none

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26120)

### CODE REVIEW

File: src/Core/Grid/Definition/Factory/RequestSqlGridDefinitionFactory.php [hunk 1]
```diff
@@ -107,6 +107,7 @@ protected function getColumns()
                         'actions' => (new RowActionCollection())
                             ->add(
                                 (new LinkRowAction('export'))
+                                    ->setName($this->trans('Export', [], 'Admin.Global'))
```
Comment (Julievrz) [inline src/Core/Grid/Definition/Factory/RequestSqlGridDefinitionFactory.php]:
```suggestion
                                    ->setName($this->trans('Export', [], 'Admin.Actions'))
```

Comment (boubkerbribri) [reply src/Core/Grid/Definition/Factory/RequestSqlGridDefinitionFactory.php]: done thanks

**Review by Julievrz — APPROVED** [review]
Wording ✅

**Review by matthieu-rolland — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @boubkerbribri

**Pull request merged**